City-of-Helsinki/helsinki-paatokset
# PP-277: Handle missing case titles

This PR adds functionality for fetching a missing case title from their latest decision. Some cases created before 2019 are missing their titles because of content quality issues.

**To test**
- Checkout branch, run `make drush-cr`
- Run these commands to import a case and a decision: 
```
drush ap:update cases HEL-2018-012998 -v
drush ap:update decisions HEL-2018-012998 -v
```
- Check the imported case: https://helsinki-paatokset.docker.so/fi/asia/hel-2018-012998
- The full title field should be empty and the node title should be "NO TITLE"
- Update the case title: `drush ahjo-proxy:update-cases --logic=title -v`
- Reload the page. The title should be picked up from the decision
- There should be a notice below the title about it missing originally
